### DISCUSSION TITLE
Silbury Hill Sunset Azimuth on 1. September 3500BC vs 2024AD has difference of 16 degrees

### DISCUSSION THREAD

**Discussion by electroreceptive1970** [Stellarium/stellarium#4017] [Q&A] [answered]
Silbury Hill Sunset Azimuth on 1. September 3500BC is 299.8.
Silbury Hill Sunset Azimuth on 1. September 2024AD is 283.3.
I cannot believe in such big difference.

From which year Stellarium works properly?
1500 AD?
1800 AD?

**Comment by gzotti** [comment, marked as answer]
There was no day called September 1, 3500BC for people living in 3500BC, so the problem was not relevant for them.

As we all know (do we?), before the Gregorian reform the Calendar was a mess. This is why we use the Gregorian calendar today. Read Appendix F in the User Guide for more information.

**Comment by gzotti** [comment]
To be precise, this date comparison at this location is permitted after September 1752 only, i.e. 1753AD September 1 was the earliest date to compare to, historically.

**Reply by lulunac27a** [reply]
The Gregorian Calendar was introduced on October 15, 1582, which was October 5, 1582, in Julian Calendar.

**Reply by gzotti** [reply]
Silbury Hill is in England. Please investigate the date of introduction of the New Calendar in Great Britain.
